src/dotVariant.Generator/templates/IObservable.scriban-cs

```diff
@@ -15,6 +15,10 @@ end
 namespace {{ Options.ExtensionClassNamespace }}
 {
 {{~ end ~}}
+    /// <summary>
+    /// Extensions which allow for easy and powerful integration into `System.Reactive.Linq`-like queries
+    /// on `IObservable&lt;T&gt;` sequences, that let you manipulate an asynchronous stream of variants based on the contained type.
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// Extensions which allow for easy and powerful integration into `System.Reactive.Linq`-like queries
    /// on <see cref="global::System.IObservable{T}" /> sequences, that let you manipulate an asynchronous stream of variants based on the contained type.
    /// </summary>
```

Reply:
actually makes sense to use "see" here(I definitely did not just copy&paste that summary from somewhere /s)